jwst/datamodels/schemas/pathlosscorr.schema.yaml

```diff
@@ -10,6 +15,11 @@ properties:
     fits_hdu: WAVELENGTH_POINTSOURCE
     ndim: 1
     datatype: float32
+  pathloss_uniformsource2d:
+    title: 2-d array for pathloss (uniform source)
+    fits_hdu: PATHLOSS_UNIFORMSOURCE2D
```

Comment:
For the ease of folks interfacing to these specific extensions, is there a way we could trim down the length of their names, while still retaining clarity? For example, `PATHLOSS_POINT` and `PATHLOSS_UNIFORM`?

Reply:
Still need the 2d part as the 1-d is also there.  Not sure it's worth taking out the SOURCE part, it is what the POINT and UNIFORM adjectives describe...